frontend/app/controllers/spree/products_controller.rb

```diff
@@ -15,7 +15,10 @@ def index
     end
 
     def show
-      @variants = @product.variants_including_master.active(current_currency).includes([:option_values, :images])
+      @variants = @product.variants_including_master
+                          .spree_base_scopes
+                          .active(current_currency)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.